### ISSUE TITLE
Make Setup Config instruction easier for new users

### ISSUE THREAD

**Issue by shyim** [muety/wakapi#325]
https://github.com/muety/wakapi/blob/8f87c4e2831dacaaae0119c3bb1866f7c0ca977b/views/summary.tpl.html#L189-L197

I setup it a few minutes ago and didn't understand what  `# Set <em>api_url = <span class="with-url-inner">%s/api</span></em><br>` means as I didn't had the file.

 Maybe I would rewrote it to something like.

Setup your local `~/.wakatime.cfg` to following:

```ini
[settings]
api_key = {{ API_KEY }}
api_url = {{ API_URL }}
```

The placeholders ofcourse replaced. I guess so would more people understand what do to 😅

**Comment by muety** [comment]
Good point. But the placeholders were properly replaced for you? So it at least looked like this?
![image](https://user-images.githubusercontent.com/8978843/156126330-80a31107-c2fc-4fe7-9299-37d9a491b4bb.png)

**Comment by shyim** [comment]
Thanks :)
